resources/commands.js

```diff
@@ -17,6 +17,7 @@ I18n.translations = {
         validation_title: 'Amount',
         validation_amount_specified: 'Amount must be specified',
         validation_invalid_number: 'Amount is not valid number',
+        validation_amount_is_too_small: 'Amount is too small',
```

Comment:
how about "Amount is too precise. The smallest unit you can send is 1 Wei (1x10^-18)"?